### PR TITLE
Random fixes in preparation for build tool.

### PR DESCRIPTION
**Pull request by mcarcaso**
- Naming some refinements
- Adding some requires
- Splitting up the net stuff into a node and web script
- Wrapping some more code with foam.SCRIPT

### CODE REVIEW

File: src/lib/netNode.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/**
```
Comment (kgrgreer) [inline src/lib/netNode.js:1]: Wrong license.

Comment (mcarcaso) [reply src/lib/netNode.js:1]: Fixed

File: src/foam/core/stdlib.js [hunk 2]
```diff
@@ -756,7 +756,7 @@ foam.LIB({
     function is(a, b) { return a === b; },
     function isInstance(o) {
       return typeof o === 'object' && ! Array.isArray(o) &&
-          ! foam.core.FObject.isInstance(o);
+          ! foam.core.FObject.isInstance(o) && ! foam.Null.isInstance(o);
```
Comment (adamvy) [inline src/foam/core/stdlib.js]: Let's do this in a different PR :)

Comment (mcarcaso) [reply src/foam/core/stdlib.js]: Done
